### PR TITLE
Clean up Generate(), part 2

### PR DESCRIPTION
**Pull request by valadaptive**
Cleans up more of the code paths and control flow in the `Generate` function and `StreamingProcessor`.

Has lots of whitespace changes; use "Hide whitespace" and go one commit at a time when reviewing.

### CODE REVIEW

File: public/script.js [hunk 1]
```diff
@@ -3377,662 +3361,639 @@ async function Generate(type, { automatic_trigger, force_name2, quiet_prompt, qu
...
-                                const regex = new RegExp(`\\b(${blacklist.join('|')})\\b`, 'gi');
-                                const matches = getMessage.match(regex) || [];
-                                return matches.length >= threshold;
-                            }
+                    // regenerate with character speech reenforced
+                    // to make sure we leave on swipe type while also adding the name2 appendage
+                    delay(1000).then(async () => {
+                        // The first await is for waiting for the generate to start. The second one is waiting for it to finish
+                        const result = await await Generate(type, { automatic_trigger, force_name2: true, quiet_prompt, skipWIAN, force_chid, maxLoops: maxLoops - 1 });
+                        return result;
```
Comment (Cohee1207) [inline public/script.js]: I think the result here may be lost. It returns to the delay callback, so the result of the circuit breaker run will not be accounted for.

Comment (valadaptive) [reply public/script.js]: Changed it to just `await` the delay

**Comment by Cohee1207** [comment]
I fixed OpenAI, but Horde is possibly broken right now

**Comment by Cohee1207** [comment]
I think I got them all. Tried all APIs streaming + no streaming + quiet gens.

**Pull request merged**